services/121-service/src/secrets.ts.example

```diff
@@ -9,3 +9,12 @@ export const USERCONFIG = {
   emailAidWorker: 'aidworker@example.org',
   countryIdAidWorker: 1,
 }
+
+
+export const TWILIO = {
+         sid: 'AC<your-sid>',
+         authToken: 'AC<your-sid>',
+         testToNumber: '<your-testToNumber>',
+         testFromNumber: '<your-testFromNumber>',
+         ngrok: 'http://bf235b61.ngrok.io/',
```

Comment:
It wouldn't _really_ hurt, but this should also not be committed, I think.

Reply:
Yes that url expires after 6 hours

Reply:
I changed it to http://example.ngrok.io/